pypxe/tftp.py

```diff
@@ -140,7 +148,6 @@ def listen(self):
             #Resent those that have timed out
             for i in self.ongoing:
                 if self.ongoing[i]['timeout']+5 < time.time() and self.ongoing[i]['retries']:
-                    print self.ongoing[i]['handle'].tell()
```

Comment:
@pbertera this is that line you caught last time that @psychomario said could be removed, right?